tests/kafkatest/tests/core/consume_bench_test.py

```diff
@@ -44,7 +44,6 @@ def setUp(self):
         self.trogdor.start()
         if self.zk:
             self.zk.start()
-        self.kafka.start()
```

Comment:
Is there a way to avoid having to remove this? It seems that we were able to do something for the `metadata_quorum` param.

Reply:
this is kinda a limitation of ducktape, where the setup call doesn't directly get any of the parameters of the test, meaning we can't know for sure here directly what the args are via parameters. However how metadata team around this was pulling the args directly from the test context (which we could do here).  I feel like its a bit less hacky to just move the start call as setup isn't all that useful imo (you can write your own setup with the args you want and call it at the start of test to achieve a similar result). but either way will work.

Reply:
I would prefer to not change this because we would need to change it in all the tests.